client/verta/tests/test_deployment.py

```diff
@@ -578,6 +578,46 @@ def test_dataframe(self, experiment_run, model_for_deployment):
         data_csv = experiment_run.get_artifact("train_data").read()
         assert X_train.join(y_train).to_csv(index=False) == six.ensure_str(data_csv)
 
+    def test_histogram(self, experiment_run):
+        np = pytest.importorskip("numpy")
+        pd = pytest.importorskip("pandas")
+
+        df = pd.concat(
+            objs=[
+                pd.Series([True]*10 + [False]*20, name='binary col'),
+                pd.Series([0]*5 + [1]*10 + [2]*15, name='discrete col'),
+                pd.Series(range(30), name='continuous col'),
+            ],
+            axis='columns',
+        )
+        X = df[['binary col', 'discrete col']]
+        y = df['continuous col']
+
+        histograms = {
+            'features': {
+                'binary col': {'histogram': {'binary': {'count': [20, 10]}}, 'type': 'binary'},
+                'discrete col': {'histogram': {'discrete': {
+                    'bucket_values': [0, 1, 2],
+                    'count': [5, 10, 15]}}, 'type': 'discrete'},
+                'continuous col': {'histogram': {'float': {
+                    'bucket_limits': [0, 2.9, 5.8, 8.7, 11.6, 14.5, 17.4, 20.3, 23.2, 26.099999999999998, 29],
+                    'count': [3, 3, 3, 3, 3, 3, 3, 3, 3, 2]}}, 'type': 'float'},
+            },
+            'total_count': 30,
+        }
+
+        experiment_run.log_training_data(X, y)
+        endpoint = "{}://{}/api/v1/monitoring/data/references/{}".format(
```

Comment:
Can you make this an unit test too? Just relying on the external system could indicate an issue in the service and not the code.